packages/store/src/awareness.ts

```diff
@@ -43,35 +44,27 @@ export type AwarenessState<
   response?: Response[];
 };
 
-interface AwarenessMessage<
+interface AwarenessEvent<
   Flags extends Record<string, unknown> = BlockSuiteFlags
 > {
   id: number;
   type: 'add' | 'update' | 'remove';
-  state?: AwarenessState<Flags>;
+  state?: RawAwarenessState<Flags>;
 }
 
-export interface AwarenessMetaMessage<
```

Comment:
@Himself65 I removed this interface since looks like it's not being used. Subjectively, the `MetaMessage` naming also makes it a bit too opaque for me to understand.